share/locale/languages.xml

```diff
@@ -37,6 +37,7 @@
   <language code="it" name="Italiano" />
   <language code="ja" name="日本語" />
   <language code="ka" name="ქართული" />
+  <language code="kab" name="ⵜⴰⵇⴱⴰⵢⵍⵉⵜ" />

```

Comment:
I would probably be better to use one of the transliteration https://en.wikipedia.org/wiki/Kabyle_language There is only one string translated on transifex but it "looks" a bit like  "Tamaziɣt Taqbaylit"